types/config.d.ts

```diff
@@ -41,12 +44,13 @@ declare global {
         path: string;
         options?: {};
       } | {
-        implementation: typeof Gatherer;
+        implementation: ClassOf<Gatherer.GathererInstance>;
```

Comment:
What does changing this do? `typeof Gatherer` already is a `ClassOf<Gatherer.GathererInstance>`

Reply:
see https://github.com/GoogleChrome/lighthouse/pull/11748#discussion_r535709713